### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by aulorbe**
Readying for merge :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,167 @@
...
 
-## What to expect
+**Noise**
 
-## Available data for public use
+To model the 'noise' that one would expect to see in real-world elections, we created elections with a certain 
+amount of noise. We defined this noise as a random number between 1-15 that would represent the percentage of total 
+ballots 
+in a specific election that would be missing a vote for a certain candidate. For example, if there were 4 possible 
+candidates for an election, a 'noisy' ballot would be one that had votes for 2/4 candidates, instead of 4/4.
```
Comment (SherinV) [inline README.md]: Maybe, we should mention somewhere that a noisy vote is a partial vote.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,167 @@
...
+Choice Voting (RCV) system, a candidate whom *no one* felt strongly about could win. 
+
+This phenomenon incorporates 
+various theorems (including, most famously, [Arrow's Impossibility Theorem](https://en.wikipedia.org/wiki/Arrow's_impossibility_theorem)), but generally goes by the name 'The Spoiler Effect.' She and this project's 
+other teammates wanted to know if this 'spoiler effect' was something voters should worry about, or if it 
+was so rare as to not affect the outcome of most elections.
+
+Of course, this project is still a work in progress years after the 2019 election. This is because a secondary 
+goal of this project is to simply increase our data science & software engineering skills. So, we have been taking our 
+time with each step in order to maximize learnings along the way (and, of course, we are doing this in our 'free' time).
```
Comment (Anxhela21) [inline README.md]: Not sure if we need lines 20-22? Or we can simplify to say "we did this for fun on our free time".

File: README.md [hunk 3]
```diff
@@ -1,32 +1,167 @@
...
 
 ## Methods
-### Obstacles & how we dealt with them:
-- Exhausted ballots: https://democracyjournal.org/arguments/ranked-choice-voting-is-not-the-solution/
-- Noise
-- Probabilities of vote spread across possible candidates
+### Data & Features
+Finding data with which to train our model with was *difficult*. There are hardly any public datasets that include 
+1+ spoiled elections (in an RCV system), let alone tens of thousands. For this reason, we chose one of the most famous 
+spoiled, RCV elections off of which we could model synthetic data: [Burlington's 2009 mayoral election.](http://bolson.org/~bolson/2009/20090303_burlington_vt_mayor.html) In this election, a candidate named Kurt Wright acted as a Spoiler.
```
Comment (Anxhela21) [inline README.md]: would this read clearer? :

For this reason, we chose one of the most famous 
spoiled, RCV elections as a model for our synthetic data:

File: README.md (hunk 1 above)
Comment (Anxhela21) [reply README.md]: partial and/or improper 

ex. partial meaning they did not vote for every round while improper could be they voted all 1's or something for all candidates (did not rank)

File: README.md [hunk 4]
```diff
@@ -1,32 +1,167 @@
...
+candidates for an election, a 'noisy' ballot would be one that had votes for 2/4 candidates, instead of 4/4.
 
-## Thanks yous
+**Number of Candidates**
 
-## Notes (will go away soon)
+After inspecting the number of candidates in various RCV elections in the past, we found that most elections had 
+between 3-8 candidates, while the simplest type of election to model the Spoiler Effect would be an election with 3 
+candidates. So, our script picks a number of candidates for each election between 3-8, with 3 candidates being the 
+most likely choice & 8 being the least.
```
Comment (Anxhela21) [inline README.md]: *just reminder we might have to update this if we choose to look at 15 candidate elections with new data

File: README.md [hunk 5]
```diff
@@ -1,32 +1,167 @@
...
+<p align="center">
+<img src="./img/condorcet_matrix.png" width="420" height="380"/>
+</p>
+
+You can see that a `1` indicates the winning candidate, and a `0` indicates the losing candidate. This matrix 
+represents a single ballot. In order to calculate the Condorcet winner across an entire election, you simply add 
+each ballot's Condorcet matrix together & take the candidate who won the highest number of head-to-head/pairwise 
+contests. 
+
+This is precisely how built our algorithm to determine whether an election was spoiled: calculate the
```
Comment (Anxhela21) [inline README.md]: This is precisely how **we**

File: README.md (hunk 1 above)
Comment (SherinV) [reply README.md]: But we are not creating improper votes in our model, are we? Those that are all the same rank..?

Comment (aulorbe) [reply README.md]: Yeah I don't think we're creating "improper" ballot (yet) @Anxhela21

Comment (aulorbe) [reply README.md]: I'll add the partial stuff!

File: README.md (hunk 2 above)
Comment (aulorbe) [reply README.md]: Okay yes, good idea :) Keep it short n' simple!

File: README.md (hunk 3 above)
Comment (aulorbe) [reply README.md]: Oo yes, I like it, thanks!

File: README.md (hunk 4 above)
Comment (aulorbe) [reply README.md]: OO yes ,thank you

**Pull request merged**